www/src/components/navigation.js

```diff
@@ -1,12 +1,14 @@
 /** @jsx jsx */
 import { jsx } from "theme-ui"
 import { useColorMode } from "theme-ui"
-import GithubIcon from "react-icons/lib/go/mark-github"
-import TwitterIcon from "react-icons/lib/fa/twitter"
+import { GoMarkGithub as GithubIcon } from "react-icons/go"
+import {
+  FaTwitter as TwitterIcon,
+  FaDiscord as DiscordIcon,
+} from "react-icons/fa"
 
 import Link from "./localized-link"
 import SearchForm from "./search-form"
-import DiscordIcon from "./discord"
```

Comment:
Can we delete https://github.com/gatsbyjs/gatsby/blob/master/www/src/components/discord.js file completely now? I think so

Reply:
Yes! it's useless.

Reply:
I will delete it then after I finish my testing - making sure new import style as not impacting bundle sizes (I understand that the change is needed when switching to `react-icons@3`, just need to make sure we suddenly won't end up in full `react-icons` package being bundled)